### PR TITLE
feat(api): add RBAC import handler

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/api/rbac/dao_rbac_global.go [hunk 1]
```diff
@@ -51,44 +52,64 @@ func insertRbacGlobalGroup(ctx context.Context, db gorpmapper.SqlExecutorWithTx,
...
-			log.Error(ctx, "rbac.getAllRBACGlobalUsers> rbac_global_users %d data corrupted", rbacUsers.ID)
-			continue
+	// Get rbac_global_users
+	rbacGlobalUsers, err := loadRbacGlobalUsersByUserID(ctx, db, userID)
+	if err != nil {
+		return false, err
+	}
+
+	// Deduplicate rbac_global.id
+	mapRbacGlobalID := make(map[int64]struct{})
```
Comment (richardlt) [inline engine/api/rbac/dao_rbac_global.go:71]: using Int64Slice from the sdk you can append every ids then use the unique func

File: engine/api/rbac/dao_rbac_global_group.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+func loadRBACGlobalGroups(ctx context.Context, db gorp.SqlExecutor, rbacGlobal *rbacGlobal) error {
+	q := gorpmapping.NewQuery("SELECT * FROM rbac_global_groups WHERE rbac_global_id = $1").Args(rbacGlobal.ID)
+	rbacGlobalGroups, err := getAllRBACGlobalGroups(ctx, db, q)
+	if err != nil {
+		return err
+	}
+	for _, rgg := range rbacGlobalGroups {
+		rbacGlobal.RBACGroupsIDs = append(rbacGlobal.RBACGroupsIDs, rgg.RbacGlobalGroupID)
```
Comment (richardlt) [inline engine/api/rbac/dao_rbac_global_group.go:57]: Reset the RBACGroupsIDs before the loop

File: engine/api/rbac/dao_rbac_global_user.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+func loadRBACGlobalUsers(ctx context.Context, db gorp.SqlExecutor, rbacGlobal *rbacGlobal) error {
+	q := gorpmapping.NewQuery("SELECT * FROM rbac_global_users WHERE rbac_global_id = $1").Args(rbacGlobal.ID)
+	rbacUserIDS, err := getAllRBACGlobalUsers(ctx, db, q)
+	if err != nil {
+		return err
+	}
+	rbacGlobal.RBACUsersIDs = make([]string, 0, len(rbacUserIDS))
+	for _, rbacUsers := range rbacUserIDS {
+		rbacGlobal.RBACUsersIDs = append(rbacGlobal.RBACUsersIDs, rbacUsers.RbacGlobalUserID)
```
Comment (richardlt) [inline engine/api/rbac/dao_rbac_global_user.go:26]: reset RBACGroupsIDs before the loop

**Review by richardlt — CHANGES_REQUESTED** [review]

File: engine/api/rbac/dao_rbac_global_user.go (hunk 3 above)
Comment (sguiheux) [reply engine/api/rbac/dao_rbac_global_user.go:26]: There is no RBACGroupsIDsn and RBACUsersIDs is already reset :)

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17845.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17845.1 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✘

**Review by yesnault — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

**Review by fsamin — APPROVED** [review]

**Pull request merged**